tests/meson.build

```diff
@@ -144,6 +144,16 @@ if gtest_dep.found()
 
     test('unittest_query', unittest_query, env: testenv)
 
+    # Run unittest_edge
+    unittest_edge = executable('unittest_edge',
+      join_paths('nnstreamer_edge', 'unittest_edge.cc'),
+      dependencies: [nnstreamer_unittest_deps],
+      install: get_option('install-test'),
+      install_dir: unittest_install_dir
+    )
```

Comment:
You will need a ```if ... endif``` for unitest_edge so that it's activated only when "edge" feature is enabled. (refer to meson_options.txt and its usages in meson.build)

Reply:
Updated.